src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -107,18 +113,18 @@ export default function transformForSubmit(formConfig, form) {
     // If certifier is also applicant, we don't need to fill out the
     // bottom "Certification" section of the PDF:
     certification:
-      transformedData?.certifierRelationship?.relationshipToVeteran
-        ?.applicant === true
+      transformedData?.certifierRole === 'applicant'
```

Comment:
Now that we have the `certifierRole` property, we no longer use the `certifierRelationship` object for this logic.